gravity-bridge/assetlist.json

```diff
@@ -0,0 +1,25 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_id": "gravity-bridge-1",
+    "assets": [
+      {
+        "description": "The native token of Gravity Bridge",
+        "denom_units": [
+          {
+            "denom": "ugraviton",
+            "exponent": 0,
+            "aliases": []
+          },
+          {
+            "denom": "graviton",
+            "exponent": 6,
+            "aliases": []
+          }
+        ],
+        "base": "ugraviton",
+        "name": "Graviton",
+        "display": "graviton",
+        "symbol": "GRAV"
```

Comment:
you can add the logo URI line with this value https://raw.githubusercontent.com/Gravity-Bridge/Gravity-Docs/main/assets/Graviton-Blue.svg

Reply:
done!